src/services/state-persistor/utils/FeatureStyleHelper.ts

```diff
@@ -0,0 +1,911 @@
+   * @return {number} Opacity.
+   * @export
+   */
+  public getOpacityProperty(feature: Feature) {
+    return this.getNumber(feature, ngeoFormatFeatureProperties.OPACITY)
+  }
+
+  /**
+   * @param {!ol.Feature} feature Feature.
+   * @return {boolean} Show measure.
+   * @export
+   */
+  public getShowMeasureProperty(feature: Feature) {
+    let showMeasure = feature.get(ngeoFormatFeatureProperties.SHOW_MEASURE)
+    if (showMeasure === undefined) {
+      showMeasure = false
+    } else if (typeof showMeasure === 'string') {
+      showMeasure = showMeasure === 'true' ? true : false
+    }
+    console.assert(typeof showMeasure === 'boolean')
+    return showMeasure
+  }
+
+  /**
+   * @param {!ol.Feature} feature Feature.
+   * @return {boolean} Show feature label.
+   * @export
+   */
+  public getShowLabelProperty(feature: Feature) {
+    let showLabel = feature.get(ngeoFormatFeatureProperties.SHOW_LABEL)
+    if (showLabel === undefined) {
+      showLabel = false
+    } else if (typeof showLabel === 'string') {
+      showLabel = showLabel === 'true' ? true : false
+    }
+    console.assert(typeof showLabel === 'boolean')
+    return showLabel
+  }
+
+  /**
+   * @param {!ol.Feature} feature Feature.
+   * @return {number} Size.
+   * @export
+   */
+  public getSizeProperty(feature: Feature) {
+    return this.getNumber(feature, ngeoFormatFeatureProperties.SIZE)
+  }
+
+  /**
+   * @param {!ol.Feature} feature Feature.
+   * @return {number} Stroke.
+   * @export
+   */
+  public getStrokeProperty(feature: Feature) {
+    return this.getNumber(feature, ngeoFormatFeatureProperties.STROKE)
+  }
+
+  // === OTHER UTILITY METHODS ===
+
+  /**
+   * @param {!ngeox.style.TextOptions} options Options.
+   * @return {!ol.style.Text} Style.
+   * @private
+   */
+  public createTextStyle_(options: { [key: string]: any }) {
+    if (options.angle) {
+      const angle = options.angle !== undefined ? options.angle : 0
+      const rotation = (angle * Math.PI) / 180
+      options.rotation = rotation
+      delete options.angle
+    }
+
+    options.font = ['normal', `${options.size || 10}pt`, 'Arial'].join(' ')
+
+    if (options.color) {
+      options.fill = new olStyleFill({ color: options.color || [0, 0, 0, 1] })
+      delete options.color
+    }
+
+    options.stroke = new olStyleStroke({
+      color: [255, 255, 255, 1],
+      width: options.width || 3,
+    })
+    delete options.width
+
+    return new olStyleText(options)
+  }
+
+  /**
+   * Get the measure of the given feature as a string. For points, you can format
+   * the result by setting a filter to apply on the coordinate with the function
+   * {@link ngeo.misc.FeatureHelper.prototype.setPointFilterFn}.
+   * @param {!ol.Feature} feature Feature.
+   * @return {string} Measure.
+   * @export
+   */
+  public getMeasure(feature: Feature) {
+    const geometry = feature.getGeometry()
+    console.assert(geometry, 'Geometry should be truthy')
```

Comment:
are we really going to keep all the `console.asserts`?

Reply:
not sure ?